examples/preapproval/change_preapproval_status.rb

```diff
@@ -0,0 +1,26 @@
+require_relative "../boot"
+
+# Change Subscription Status
+#
+# You need to set:
+#   - AccountCredentials (EMAIL, TOKEN)
+#   - Subscription's code
+#   - The new status
+#
+# P.S: Take a look at the boot file example for more details
+
+credentials = PagSeguro::AccountCredentials.new('user@example.com', 'TOKEN')
+code = 'CODE'
+status = :suspended # Now just available the status :active and :suspended
```

Comment:
I like this way, I mean: we can put all variable on the top of the file:

```
user = "user@example.com"
token = 'TOKEN'
code = 'CODE'
status = :suspended # Now just available the status :active and :suspended

credentials = PagSeguro::AccountCredentials.new(user, token)
change = PagSeguro::SubscriptionChangeStatus.new(code, status)
# […]
 ```

It will be easy to identify that to change in the file :)

:clap: Good ideia @rafaelrpbelo